.circleci/defaults.sh

```diff
@@ -79,4 +79,4 @@ mapping["firesim-multiclock"]="SCALA_TEST=firesim.firesim.RocketMulticlockF1Test
 mapping["fireboom"]="SCALA_TEST=firesim.firesim.BoomF1Tests"
 mapping["icenet"]="SUB_PROJECT=icenet"
 mapping["testchipip"]="SUB_PROJECT=testchipip"
-
+mapping["chipyard-sodor"]="CONFIG=Sodor5StageConfig"
```

Comment:
Can you order this with the cores? Also you need to add `chipyard-sodor` to the `groups` map.

Reply:
Done, waiting for test